src/platform/platform_worker.c

```diff
@@ -128,14 +128,13 @@ CxPlatWorkersLazyStart(
     }
 
     const uint16_t* ProcessorList;
-    UNREFERENCED_PARAMETER(Config); // TODO - use config
-    /*if (Config && Config->ProcessorCount) {
+    if (Config && Config->ProcessorCount) {
         CxPlatWorkerCount = Config->ProcessorCount;
         ProcessorList = Config->ProcessorList;
-    } else {*/
+    } else {
         CxPlatWorkerCount = CxPlatProcMaxCount();
         ProcessorList = NULL;
-    //}
+    }
```

Comment:
More changes are going to be necessary for this stuff I expect. I have a WIP PR related to this already open, and it's really complicated.